### DISCUSSION TITLE
Link to collection

### DISCUSSION THREAD

**Discussion by Limezy** [outline/outline#3484] [Ideas]
I would like to be able to link to a collection.
This would be useful for instance when building a welcome page in the wiki to redirect people to different collections.

**Comment by tommoor** [comment]
Some auto-complete would be nice, in the meantime you can link to a collection like any other manual link of course

**Reply by Limezy** [reply]
That's right, it works well indeed.
I also just found out that we could link to a specific header in a document ! That's great, but it would be nice to have : 
* A kind of auto-complete for that too
* Hover preview showing that header instead of the beginning of the document
